src/Config/SessionConfig.php

```diff
@@ -246,6 +263,27 @@ public function setHashBitsPerCharacter($hashBitsPerCharacter)
     }
 
     /**
+     * Set session.sid_bits_per_character
+     *
+     * @param  int $sidBitsPerCharacter
+     * @return SessionConfig
+     * @throws Exception\InvalidArgumentException
+     */
+    public function setSidBitsPerCharacter($sidBitsPerCharacter)
+    {
+        if (!is_numeric($sidBitsPerCharacter)
+            || !in_array($sidBitsPerCharacter, $this->validSidBitsPerCharacters)
```

Comment:
`||` is correct:
```php
if ($cond1 && $cond2) {
    ok_code();
}
error();
// Otherwise
if (! $cond1 || ! $cond2) {
    error();
}
ok_code();
```
In merit of `in_array` strict parameter, althought I would also advocate the adoption of strictness, for ex. with the [PHP-CS-Fixer\StrictParamFixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.3.2/src/Fixer/Strict/StrictParamFixer.php), we can keep it like this to stay even with [setHashBitsPerCharacter](https://github.com/zendframework/zend-session/blob/release-2.7.3/src/Config/SessionConfig.php#L234-L246) method (that I just copy-pasted) and because `is_numeric` already ensure the check.

Otherwise we can go with strict param and remove `is_numeric` call and optain the same result; in this case I would also go with chainging the all `in_array` occurrencies to keep a standard.

Let me know what maintainers prefer.